cpp/src/arrow/json/chunker-test.cc

```diff
@@ -49,9 +49,9 @@ std::string join(Lines&& lines, std::string delimiter) {
   return joined;
 }
 
-std::string PrettyPrint(std::string one_line) {
```

Comment:
While debugging this I found that GCC 4.8 was not making a deep copy of this argument, which resulted in the corruption of test data for tests following `ChunkerTest.PrettyPrinted`

Reply:
I guess the implementation was copy-on-write?